rcsb/utils/io/StashUtil.py

```diff
@@ -83,6 +83,10 @@ def storeBundle(self, url, remoteDirPath, remoteStashPrefix="A", userName=None,
                 if ok:
                     remotePath = os.path.join("/", remoteDirPath, fn)
                     ok = sftpU.put(self.__localStashTarFilePath, remotePath)
+            elif url.startswith("/") or url.startswith("file"):
```

Comment:
If you decide to enforce `file://`, then I would be explicit here. Also down below.